app/models/index.js

```diff
@@ -6,7 +6,9 @@ const fs = require('fs'),
   dbConfig = require('../../config/db')[config.environment],
   db = {};
 
-const sequelize = new Sequelize(dbConfig.database, dbConfig.username, dbConfig.password, dbConfig);
+const sequelize = process.env.DATABASE_URL
```

Comment:
why is DATABASE_URL coming directly from process.env rather than a property in the config?